### ISSUE TITLE
warning: destination for mariadb.networkPolicy.egressRules.customRules is a table. Ignoring non-table value ([])

### ISSUE THREAD

**Issue by ametad** [bitnami/charts#15821]
### Name and Version

bitnami/wordpress 15.2.59

### What architecture are you using?

amd64

### What steps will reproduce the bug?

Sinds this commit: https://github.com/bitnami/charts/commit/a4ba117fdc8a9be6eb5ae83f0e48ed029ab9407e

When I upgrade my wordpress Helm deployment, I get this error:

```
coalesce.go:223: warning: destination for mariadb.networkPolicy.egressRules.customRules is a table. Ignoring non-table value ([])
```

Before that, no problems at all. And I did not change values, just wanted to go the the most recent bitnami/wordpress (and sub charts) Chart version(s).

### Are you using any custom parameters or values?

Yes, here are the redacted (######) values:

```yaml
allowOverrideNone: false
autoscaling:
  enabled: true
  maxReplicas: 10
  minReplicas: 2
  targetCPU: 80
  targetMemory: null
global:
  storageClass: cvo-nas-cm
htaccessPersistenceEnabled: true
ingress:
  annotations:
    cert-manager.io/cluster-issuer: letsencrypt-production
    kubernetes.io/ingress.allow-http: "true"
    nginx.ingress.kubernetes.io/configuration-snippet: |
      if ($host = 'www.######' ) {
        rewrite ^ https://######$request_uri permanent;
      }
    nginx.ingress.kubernetes.io/global-rate-limit: "600"
    nginx.ingress.kubernetes.io/global-rate-limit-ignored-cidrs: ######
    nginx.ingress.kubernetes.io/global-rate-limit-key: ${remote_addr}-${proxy_add_x_forwarded_for}
    nginx.ingress.kubernetes.io/global-rate-limit-window: 1m
  enabled: true
  extraHosts:
    - name: www.######
      path: /
  extraTls:
    - hosts:
      - ######l
      - www.######
      secretName: ######-extra-tls
  hostname: ######
  ingressClassName: nginx
  tls: true
mariadb:
  primary:
    resources:
      limits: {}
      requests:
        cpu: 50m
        memory: 256Mi
memcached:
  enabled: true
  resources:
    limits: {}
    requests:
      cpu: 15m
      memory: 100Mi
persistence:
  accessModes:
  - ReadWriteMany
  size: 10Gi
replicaCount: 1
resources:
  limits: {}
  requests:
    cpu: 75m
    memory: 512Mi
service:
  type: ClusterIP
wordpressConfigureCache: true
wordpressPassword: ######
wordpressScheme: https

```

### What is the expected behavior?

Just a successful helm upgrade without warnings.

### What do you see instead?

This warning:

`coalesce.go:223: warning: destination for mariadb.networkPolicy.egressRules.customRules is a table. Ignoring non-table value ([])
`

Good news is, that the upgrade is not failed.

### Additional information

There is an update been in the MariaDB Chart that could be responsible for this: https://github.com/bitnami/charts/commit/a4ba117fdc8a9be6eb5ae83f0e48ed029ab9407e

**Comment by corico44** [comment]
We are going to review the following PR: https://github.com/bitnami/charts/pull/15638 to verify that it is causing the error. 
We will report in this issue any news about this error. Thank you very much for reporting it!

**Comment by corico44** [comment]
The bug should be fixed with the new version 11.5.7: #16033.
If any other question arises related to this error, this issue will remain open for a few more days until it is automatically closed.
